frontend/nginx/nginx.https.template

```diff
@@ -0,0 +1,92 @@
+upstream backend_proxy {
+    server ${BACKEND_PROXY} fail_timeout=0;
+}
+
+## REACT APP ## 
+server {
+    server_name     ${MAIN_DOMAIN};
+    listen          443 ssl;
+
+    # ssl
+    include /etc/nginx/snippets/ssh-params.conf;
+    ssl_certificate /etc/ssl/${CERT_FILE_NAME};
+    ssl_certificate_key /etc/ssl/${KEY_FILE_NAME};
+    ssl_dhparam /etc/ssl/${DHPARAM_FILE_NAME};
+
+    charset     utf-8;
+    client_max_body_size 75M;
+
+    # React app production build
+    location /  {
+        root /usr/share/nginx/html;
+        try_files $uri $uri/ /index.html;
+    }
+}
+
+## DJANGO API ## 
+server {
+    server_name     ${API_DOMAIN};
+    listen          443 ssl;
+
+    # ssl
+    include /etc/nginx/snippets/ssh-params.conf;
+    ssl_certificate /etc/ssl/${CERT_FILE_NAME};
+    ssl_certificate_key /etc/ssl/${KEY_FILE_NAME};
+    ssl_dhparam /etc/ssl/${DHPARAM_FILE_NAME};
+
+    charset     utf-8;
+    client_max_body_size 75M;
+
+    location / {
+        try_files $uri @app;
+    }
+    location @app {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+        proxy_redirect off;
+        proxy_pass http://backend_proxy;
+    }
+}
+
+### AUTHENTICATION ##
```

Comment:
More commented out code... :P

Reply:
Agreed! I'll save this section until we are ready for authentication